src/tests/bulkConcepts/addBulkConcepts.test.jsx

```diff
@@ -28,6 +28,7 @@ describe('Add Bulk Concepts', () => {
       isFetching: true,
       match,
       addExistingBulkConcepts: jest.fn(),
+      language: 'en',
```

Comment:
How is this related to the fix of the back button?

Reply:
yes. the link has a `language` of `en` at the end.
e.g. `http://localhost:3000/concepts/users/admin/858738987555379/CHD/en/`